### DISCUSSION TITLE
Converting YOLOv5 training dataset to YOLOv8

### DISCUSSION THREAD

**Discussion by AlainPilon** [roboflow/supervision#864] [General]
Is there an easy way to convert a dataset exported under the YOLOv5 format/structure to a YOLOv8 format/structure? 

I looked at the Supervision code and it seems like it only supports YOLOV8. The YOLOv5 method is just to load the trained model.

**Comment by SkalskiP** [comment]
Is there any difference?

**Comment by AlainPilon** [comment]
@SkalskiP Yes! For some reasons, the val/train and labels/images levels in the directory tree has been reversed. And I think the `data.yaml` file has some small changes too (not sure)

**Comment by AlainPilon** [comment]
In the end, I managed to process the dataset by specifying the `data` param to the YOLOv8 CLI and point it to the v5 `dataset.yaml` file. Not documented, but obvious once you know about it!

**Reply by SkalskiP** [reply]
Awesome! Nonetheless, I think it will be addressed at some point. At the moment we simply have too much work.
